superset/db_engine_specs/base.py

```diff
@@ -1425,6 +1430,33 @@ def cancel_query(  # pylint: disable=unused-argument
     def parse_sql(cls, sql: str) -> List[str]:
         return [str(s).strip(" ;") for s in sqlparse.parse(sql)]
 
+    @classmethod
+    def _humanize(cls, value: Any, suffix: str, category: Optional[str] = None) -> str:
```

Comment:
If there could be an input like (1000, "", "dollars") and an output like "$1,000", there would be more categories